ci/pipeline.yml

```
@@ -70,10 +70,28 @@ jobs:
       - unit-director-postgres-13
       - unit-director-postgres-15
       - run-bundle-audit
+      - brats-utils
     plan:
       - get: bosh
         trigger: true
 
+  - name: brats-utils
+    serial: true
+    serial_groups: [ unit-common ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)